src/averageAges.js

```diff
@@ -20,6 +20,26 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+
+  const allMale = people.filter(person => {
+    if (century === undefined) {
+      return person.sex === 'm';
+    } else {
+      return person.sex === 'm'
+        && (Math.floor(person.born / 100)
+        && Math.floor(person.died / 100)) === century - 1;
```

Comment:
Consider using variables with meaningful names instead of magic strings and numbers